### ISSUE TITLE
How to change background colors, marker css and add hover text on marker?

### ISSUE THREAD

**Issue by abhisheksimion** [visjs/angular-visjs#39]
I am trying to create a timeline view using visjs of a upgrade scenario (Pre Upgrade, Pre Release & Post Upgrade) something similar to the image below. Need some pointers to create different region colors as depicted in the image, css to change the main marker to an image source and also on hover of the slider (region or markers) it should show some description.

![image](https://cloud.githubusercontent.com/assets/9957912/8982717/8fadea42-36e1-11e5-9859-c34de56e2ad5.png)

I have posted a [stackoverflow question here](http://stackoverflow.com/questions/31722383/angularjs-how-to-change-background-colors-marker-css-and-add-hover-text-on-mar).

[Plunker link](http://plnkr.co/edit/38XxEVX9kbfyVYRXw0Y7?p=preview)

**Comment by josdejong** [comment]
Did you have a look at background areas? 

http://visjs.org/examples/timeline/items/backgroundAreas.html

**Comment by abhisheksimion** [comment]
@josdejong Yes I followed the same example, but somehow the background areas are not rendering. Please check [here](http://plnkr.co/edit/38XxEVX9kbfyVYRXw0Y7?p=preview)

$scope.visData = new vis.DataSet([
      {start: '2015-07-26', end: '2015-08-25', type: 'background', title: 'Pre Upgrade', className: 'preupgrade'},
      {start: '2015-08-26', end: '2015-09-30', type: 'background', title: 'Pre Release', className: 'prerelease'},
      {start: '2015-10-01', end: '2015-10-31', type: 'background', title: 'Post Upgrade', className: 'postupgrade'}
    ]);

**Comment by josdejong** [comment]
I think there is something going wrong where you load the data into the Timeline:

```js
timeline = new vis.Timeline(element[0], scope.data.items, scope.data.groups, scope.options);
```
When I log `scope.data`, it's a DataSet, not an object containing properties `items` and `groups`.

Besides that, are you actually using `angular-visjs`? Because in your example you're loading the vanilla JavaScript version of the Timeline (`new vis.Timeline(...)` instead of `<time-line data="data" options="options" events="events"></time-line>`)
